src/main/java/com/google/sps/data/Post.java

```diff
@@ -57,6 +58,14 @@ public long getTimestamp() {
         return timestamp; 
     }
 
+    public long totalReactionCount() {
```

Comment:
can this be a static helper?
It is a cleaner structure since it doesn't require state.

public static long getNumReactions(Post a) {
   ...
}